File: 001-010/009/009.js
#!/usr/bin/env node

(function Finder() {
    for (var a = 2; a < 1001; a++) {
        for (var b = 3; b < 1001; b++) {
            for (var c = 4; c < 1001; c++) {
                if (a + b + c == 1000) {
                    if (Math.pow(a, 2) + Math.pow(b, 2) == Math.pow(c, 2)) {
                        console.log(a + ", " + b + ", " + c + ", " + a * b * c);
                        return;
                    }
                } else if (a + b + c > 1000) {
                    break;
                }
            }
        }
    }
})();

File: 001-010/009/009.py
#!/usr/bin/env python


def Finder():
    for a in range(2, 1001):
        for b in range(3, 1001):
            for c in range(4, 1001):
                if a + b + c == 1000:
                    if a**2 + b**2 == c**2:
                        return a, b, c, a*b*c
                elif a + b + c > 1000:
                    break


if __name__ == "__main__":
    print(Finder())
